cmd/commands/tx.go

```diff
@@ -12,7 +12,7 @@ import (
 
 	cmn "github.com/tendermint/go-common"
 	client "github.com/tendermint/go-rpc/client"
-	"github.com/tendermint/go-wire"
+	wire "github.com/tendermint/go-wire"
```

Comment:
Why are we labelling this library? Is this just stylistic? I've noticed that sometimes our code will rename a library to it's already default name (as is done here)... is the idea just not to confuse people because the repo is 'go-wire' and not just 'wire'? If not this seems redundant... just curious as to what folks general sentiments are